core/src/main/java/org/apache/accumulo/core/spi/common/ServiceEnvironment.java

```diff
@@ -17,91 +17,27 @@
 
 package org.apache.accumulo.core.spi.common;
 
-import java.util.Iterator;
-import java.util.Map;
-import java.util.Map.Entry;
-
-import org.apache.accumulo.core.client.TableNotFoundException;
+import org.apache.accumulo.core.client.PluginEnvironment;
 import org.apache.accumulo.core.data.TableId;
 
 /**
  * This interface exposes Accumulo system level information to plugins in a stable manner. The
  * purpose of this interface is to insulate plugins from internal refactorings and changes to
  * Accumulo.
  *
+ * <P>
```

Comment:
Please use lower case html tags, as per W3C recommendations for HTML5 (and for consistency with older XHTML requirements, and because it looks less "shouty").